src/jit/gentree.h

```diff
@@ -2357,9 +2432,10 @@ struct GenTreeCall final : public GenTree
     CORINFO_SIG_INFO* callSig;                // Used by tail calls and to register callsites with the EE
 
     regMaskTP         gtCallRegUsedMask;      // mask of registers used to pass parameters
+
 #ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
-    SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR structDesc;
-#endif // FEATURE_UNIX_AMD64_STRUCT_PASSING
+    MultiRegReturnTypeDesc    gtMultiRegReturnTypeDesc;
+#endif 
```

Comment:
random side note: it seems weird that we have FEATURE_UNIX_AMD64_STRUCT_PASSING guarding struct *returning* data. Maybe this needs to be FEATURE_UNIX_AMD64_STRUCT_ABI. Maybe FEATURE_UNIX_AMD64_STRUCT_PASSING can now be retired in favor of UNIX_AMD64_ABI, since we probably (!) no longer have a need to have the Unix ABI *without* the struct passing part.